include/items.php

```diff
@@ -167,6 +168,14 @@ function add_page_info_data($data) {
 
 	$text = "[attachment type='".$data["type"]."'";
 
+	if ($data["text"] == "") {
+		$data["text"] = $data["title"];
+	}
+
+	if ($data["text"] == "") {
+		$data["text"] = $data["url"];
+	}
+
 	if ($data["url"] != "")
```

Comment:
When I do this, do you and @Hypolite promises not to add further "Since you changed that line, please change this and that according to the standards" comments? :-) I want to prevent making every small bugfix a mammoth task.